utils/fixed_decimal/src/decimal.rs

```diff
@@ -118,8 +118,8 @@ pub struct FixedDecimal {
 }
 
 /// A specification of the sign used when formatting a number.
+#[non_exhaustive]
```

Comment:
Issue: I would make this an exhaustive struct. There are only 3 sign values, and they correspond to the low-level data model of FixedDecimal and UTS 35.

Reply:
https://github.com/unicode-org/icu4x/issues/2486

Reply:
Okay, I am creating another PR to revert this.